ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -32,10 +35,21 @@ public extension Colors {
 public class FluentUIFramework: NSObject {
     @objc public static var bundle: Bundle { return Bundle(for: self) }
     @objc public static let resourceBundle: Bundle = {
+        #if SWIFT_PACKAGE
+        return Bundle.module
+        #else
         guard let url = bundle.resourceURL?.appendingPathComponent("FluentUIResources-ios.bundle", isDirectory: true), let bundle = Bundle(url: url) else {
```

Comment:
this -ios.bundle is still ok? any update we need to do for nugetpackage copying scripts if the bundle name is different? so devmain should know?

Reply:
Nothing changed for existing pipelines, so nothing should need to change. Nuget builds will not be based off SPM right now.